kuma/javascript/src/payments/index.jsx

```diff
@@ -0,0 +1,367 @@
+
+type PaymentsRouteParams = {
+    locale: string
+};
+
+export default function PaymentsLandingPage() {
+    const locale = getLocale();
+    const urls = {
+        annualReport:
+            'https://www.mozilla.org/en-US/foundation/annualreport/2019/',
+        email:
+            'mailto:mdn-support@mozilla.com?Subject=Manage%20monthly%20subscription',
+        moco: 'https://www.mozilla.org/foundation/moco/',
+        mozillaFoundation: 'https://www.mozilla.org/foundation/',
+        managePayments: `/${locale}/payments/recurring/management`,
+        stripe: 'https://stripe.com/',
+        taxDeductible: 'https://donate.mozilla.org/faq#item_tax_a',
+        terms: `/${locale}/payments/terms`
+    };
+
+    return (
+        <UserProvider>
+            <A11yNav />
+            <Header />
+            <SubHeader
+                title="Become a monthly supporter"
+                description="Support MDN with a $5 monthly subscription and get back more of the knowledge and tools you rely on for when your work has to work."
+                columnWidth="7"
+            />
+            <main
+                id="contributions-page"
+                className="contributions-page"
+                role="main"
+            >
+                <section className="section">
+                    <header>
+                        <h2>{gettext('FAQs')}</h2>
+                    </header>
+                    <ol id="contribute-faqs" className="faqs clear">
+                        <ListItem
+                            title={gettext('Why is MDN asking me for money?')}
+                            number="1"
+                        >
+                            <p>
+                                {gettext(
+                                    'MDN is seeking direct support from our users. We’d like to accelerate our growth and extend the maintenance of MDN’s content and platform, with assistance from those who use it.'
+                                )}
+                            </p>
+                            <p>
+                                {gettext(
+                                    'Our user base has grown exponentially in the last few years and we have a large list of improvements we’d like to make. While MDN is currently wholly funded by Mozilla, and has been from the beginning, we’d like to create a closer, more collaborative relationship between our audience (that’s you!), our content (written for you and sometimes by you), and our supporters (also, you, again)-- to accelerate those improvements.'
+                                )}
+                            </p>
+                        </ListItem>
+                        <ListItem
+                            title="How is my payment handled? Is it secure?"
+                            number="2"
+                        >
+                            <p>
+                                <Interpolated
+                                    id={gettext(
```

Comment:
Oh interesting, not having looked at the output, this suggest that the `<stripeLink />` bit works in plain `gettext` calls? I thought we had to use `Interpolated` for that.